src/coreclr/.nuget/pkgproj.targets

```diff
@@ -0,0 +1,76 @@
+<Project>
+  <PropertyGroup>
+    <StableVersion Condition="'$(DotNetFinalVersionKind)' == 'release'">$(ProductVersion)</StableVersion>
```

Comment:
If the eventual goal is to get rid of the thing we call "pkgproj", shouldn't these files be named something else? Or is it just a goal to shrink down what we need to do with pkgproj

Reply:
These files provide the support targets for the pkgproj-style projects that remain. Once we remove them (future PR), we can remove these files.